### PR TITLE
Remove unexpected token

### PR DESCRIPTION
**Pull request by geobde**
Fix the bug Failed to compile(develop-branch) #763

### CODE REVIEW

File: src/components/Charts/utils.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const lastValidPoint = (data: Highcharts.Point[]) =>
 export const formatDecimal = (num: number, places = 2): string =>
   num.toFixed(places);
 
-export const formatPercent = (num: number, places = 0): string =>
+export const formatPercent = (num: number, places = 1): string =>
```
Comment (mikelehen) [inline src/components/Charts/utils.ts:16]: Did you change this on purpose? Seems unrelated to the PR description. 😁

File: src/screens/CompareModels/CompareModels.js [hunk 2]
```diff
@@ -210,7 +210,7 @@ export function CompareModels({ match, location }) {
           <FormControl style={{ width: '12rem' }}>
             <InputLabel focused={false}>Sort by:</InputLabel>
             <Select value={sortType} onChange={changeSort}>
-              ><MenuItem value={SORT_TYPES.ALPHABETICAL}>State Name</MenuItem>
+              <MenuItem value={SORT_TYPES.ALPHABETICAL}>State Name</MenuItem>
```
Comment (mikelehen) [inline src/screens/CompareModels/CompareModels.js:213]: I actually just fixed this separately as a part of #756.

File: src/components/Charts/utils.ts (hunk 1 above)
Comment (geobde) [reply src/components/Charts/utils.ts:16]: Nope it was on accident.
